media/js/src/assetDetail/AssetDetail.jsx

```diff
@@ -789,8 +794,9 @@ export default class AssetDetail extends React.Component {
                     {annotationTools}
                     <div
                         id={`map-${this.props.asset.id}`}
-                        className="ol-map"
-                        aria-label={`Asset titled ${this.props.asset.title}`}></div>
+                        className="ol-map target"
```

Comment:
Is the new `target` class doing anything here?

Reply:
Ah no. I used that for testing. Will remove.